doc/project/rfc_process.md

```diff
@@ -22,17 +22,22 @@ This document outlines the basic structure of the RFC process, with the expectat
    * If requested, the TC could help to nominate a small group to shepherd the RFC.
 * If the RFC potentially contains information that could be certification-sensitive (guidance to be shared), send a note to security@opentitan.org first for feedback.
 * The RFC is shared publicly by filing a GitHub issue and tagging with the `RFC:Proposal` label.
-* Once the author is happy that the RFC is complete, they submit it to the Technical Committee for review by adding the Github label `TC Review`.
+* Once the author is happy that the RFC is complete, they submit it to the Technical Committee by converting the label to `RFC:TC Review`.
 * The Technical Committee will consider active RFCs in each meeting (those that have been marked ready for at least a week).
```

Comment:
Not part of this PR but related I guess: does "marked ready" in this context mean it has the label `RFC:TC Review`